lib/rest-adapter.js

```diff
@@ -403,6 +411,14 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  //For now: exclude paths with parameters like :id
+  //and handler functions with err obj i.e. (err, req,res,next)
+  if (!route.path.includes(':') && handler.length < 4) {
+    handler.methodName = sharedMethod.name;
+    handler.path = route.fullPath;
+    customRouter.registerPathAndHandlers({ verb: verb, fullPath: route.fullPath }, handler);
+  }
```

Comment:
Please move as much of this code into `RestRouter` as possible.

```js
if (fastRouter.canRegister(route.path, handler)) {
  fastRouter.registerPathAndHandler({
    verb: verb,
    fullPath: route.fullPath,
    methodName: sharedMethod.name
  }, handler); // << perhaps use "stringName" instead?
}
```

I am wondering, can we make the API of our new router compatible with the vanilla Express Router? Instead of `fastRouter.registerPathAndHandlers`, one would call:

```js
// function(path, handler, options)
fastRouter[verb].use(fullPath, handler, { displayName: sharedMethod.stringName });
```

I borrowed "displayName" from here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/displayName. It was initially proposed for ES6, but never made it.